src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,31 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringCalculatorTest {
+
+    private StringCalculator stringCalculator;
+
+    @ParameterizedTest
+    @DisplayName("조건에 맞는 문자열 연산 테스트")
+    @CsvSource(value = {"2 + 3 * 4 / 2 = 10", "1 + 0 = 1", "1 - 2 = -1"}, delimiter = '=')
+    void calculateTest(String value, Double expected) {
+        assertThat(stringCalculator.calculate(value)).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @DisplayName("문자열이 공백일 때 테스트")
+    @NullSource
+    void checkNullTest(String input) {
+        assertThatThrownBy(() -> stringCalculator.checkNull(input))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("empty string");
```

Comment:
```suggestion
                .hasMessage("입력 값이 공백입니다.");
```
hasMessage는 실제 Exception이 전달하는 Message를 검증하는 메소드 입니다. 
입력값으로 Null이 들어올 경우 위와같은 메시지를 전달하고 계십니다 😄